src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/commands/mobile/get_vehicle_data_request.cc

```diff
@@ -126,6 +126,12 @@ void GetVehicleDataRequest::on_event(const event_engine::Event& event) {
       std::string response_info;
       GetInfo(message, response_info);
 
+      if (message[strings::msg_params].keyExists(strings::tire_pressure)) {
+        ApplicationSharedPtr app =
```

Comment:
Does this check protocol version before populating the default values? Also instead of adding default value for missing params for earlier app versions, is it possible to include previously cached values if available?